test/test_sawtooth.py

```diff
@@ -0,0 +1,425 @@
+#!/usr/bin/env python3
+
+import datetime as dt
+import unittest
+import sys
+import warnings
+
+from examples.example_sawtooth import SawtoothWaveform
+from examples.example_sawtooth import TimestepWarning
```

Comment:
These two lines throw an error saying that the example module can not be found.

Reply:
Please share how you are running the tests. The Makefile is working currently, so I think the difference is in how the code is being called.

Reply:
Yes, that was the problem. Calling the file inside the _test_ directory causes this error to occur.